packages/expo-image-picker/expo-module.config.json

```
@@ -0,0 +1,6 @@
+{
+  "platforms": ["ios"],
```

```suggestion
  "platforms": ["ios", "android"],
```

Otherwise it's not autolinked on Android...